session/session.go

```diff
@@ -1563,7 +1564,7 @@ func (s *session) ExecuteStmt(ctx context.Context, stmtNode ast.StmtNode) (sqlex
 	logStmt(stmt, s)
 	recordSet, err := runStmt(ctx, s, stmt)
 	if err != nil {
-		if !kv.ErrKeyExists.Equal(err) {
+		if !kv.ErrKeyExists.Equal(err) && !storeerr.ErrLockAcquireFailAndNoWaitSet.Equal(err) {
```

Comment:
There is a func called `errIsNoisy` in session.go already (which checks for kv.ErrKeyExists). Maybe this can be merged in with this func and simplified?

Reply:
Done, PTAL